### PR TITLE
Create driveitem-get-SensitivityLabel.md

### PR DESCRIPTION
**Pull request by SanjoyanM**

### CODE REVIEW

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+author: SanjoyanM
+title: Get the sensitivity label attached to a file
+localization_priority: Priority
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:4]: Please assign a proper value for the localization priority: high, medium, none.

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+ms.prod: "sharepoint"
+description: "Get the sensitivity label attached to a file."
+doc_type: apiPageType
+---
+# Get sensitivity label metadata for a DriveItem resource
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:9]: I need to check further the API design to properly evaluate this title. Please bear with me while I double check with the team.

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+Namespace: microsoft.graph
+
+Retrieve the sensitivity label metadata for a [DriveItem](../resources/driveitem.md) in a [Drive](../resources/drive.md) by file system path or ID.
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:13]: Please use camel case for **driveItem** and **drive**.

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+[!INCLUDE [sample-code](../includes/snippets/javascript/get-drive-root-javascript-snippets.md)]
+[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
+
+# [Objective-C](#tab/objc)
+[!INCLUDE [sample-code](../includes/snippets/objc/get-drive-root-objc-snippets.md)]
+[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
+
+# [Java](#tab/java)
+[!INCLUDE [sample-code](../includes/snippets/java/get-drive-root-java-snippets.md)]
+[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:80]: Please remove the snippets. These are automatically generated and we should not add them manually. Just leave the API doctor code and the request code.

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    "@odata.etag": "\"{7CB88CCD-94CC-47C9-A7C8-7DFDAF0959C2},10\"",
+    "sensitivityLabel": {
+        "displayName": "Public",
+        "id": "87867195-f2b8-4ac2-b0b6-6bb73cb33afc",
+        "protectionEnabled": false
+    }
+}
+```
+
+## Remarks
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:106]: Please clarify this additional section. It is not part of the template for a reference topic.

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md (hunk 1 above)
Comment (SanjoyanM) [reply api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:4]: Done

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md (hunk 5 above)
Comment (SanjoyanM) [reply api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:106]: Removed. Please check.

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md (hunk 4 above)
Comment (SanjoyanM) [reply api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:80]: Done. Please review.

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md (hunk 3 above)
Comment (SanjoyanM) [reply api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:13]: Done.

File: api-reference/v1.0/api/driveitem-get-SensitivityLabel.md (hunk 2 above)
Comment (SanjoyanM) [reply api-reference/v1.0/api/driveitem-get-SensitivityLabel.md:9]: Based on our last conversation let us know if we are good.

**Pull request closed without merging**